src/app/helptext/system/cloudcredentials.ts

```diff
@@ -57,9 +57,14 @@ export const helptext_system_cloudcredentials = {
  target="_blank">Simple Storage Service Website Endpoints</a>.'
     )
   },
+
   region_s3: {
     placeholder: T("Region"),
-    tooltip: T(""),
+    tooltip: T('<a href="https://docs.aws.amazon.com/general/latest/gr/rande-manage.html" \
+ target="_blank">AWS resources in a geographic area</a>. Leave empty to \
+ automatically detect the correct public region for the bucket. Enter a \
+ private region name to allow the credential to find Amazon buckets \
```

Comment:
Credential does not find anything, can it be something like "Enter a private region name to allow using this credential with S3 buckets created in that region."

Reply:
Yes, I can reword that.